java/compaction-core/src/main/java/sleeper/compaction/job/status/CompactionJobStartedStatus.java

```diff
@@ -17,19 +17,26 @@
 
 import java.time.Instant;
 import java.util.Objects;
+import java.util.UUID;
 
 public class CompactionJobStartedStatus {
 
     private final Instant updateTime;
     private final Instant startTime;
+    private final String taskId;
 
-    private CompactionJobStartedStatus(Instant updateTime, Instant startTime) {
+    private CompactionJobStartedStatus(Instant updateTime, Instant startTime, String taskId) {
         this.updateTime = Objects.requireNonNull(updateTime, "updateTime may not be null");
         this.startTime = Objects.requireNonNull(startTime, "startTime may not be null");
+        this.taskId = taskId;
+    }
+
+    public static CompactionJobStartedStatus updateAndStartTimeWithTaskId(Instant updateTime, Instant startTime, String taskId) {
+        return new CompactionJobStartedStatus(updateTime, startTime, taskId);
     }
 
     public static CompactionJobStartedStatus updateAndStartTime(Instant updateTime, Instant startTime) {
-        return new CompactionJobStartedStatus(updateTime, startTime);
+        return new CompactionJobStartedStatus(updateTime, startTime, UUID.randomUUID().toString());
     }
```

Comment:
This function is only used in tests, and I'm not sure if the tests will be correct like this. If it's needed can we move it to the test code?